aqua/emacs-mac-app/Portfile

```diff
@@ -15,14 +15,15 @@ maintainers         nomaintainer
 
 description         Emacs Mac port
 
-long_description    ${name} is "Mac port" addition to GNU Emacs 25. This provides a native \
+long_description    ${name} is "Mac port" addition to GNU Emacs 26. This provides a native \
```

Comment:
You might as well replace the hardcoded version number in the `long_description` with `${emacs_version}` so this does not need to be updated going forward.

Reply:
Good idea. Done.